### PR TITLE
Steering wheel location

### PR DESCRIPTION
**Pull request by kshala-ford**
This proposal is about adding a new parameter to the `VehicleType` structure to inform the app about the steering wheel location within the vehicle.

### CODE REVIEW

**Comment by kshala-ford** [comment]
@theresalech The proposal is ready for review

File: proposals/NNNN-steering-wheel-location.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+* Status: **Review ready**
+* Impacted Platforms: [Core / iOS / Android / RPC ]
+
+## Introduction
+
+This proposal is about adding a new parameter to the `VehicleType` structure to inform the app about the steering wheel location within the vehicle.
+
+## Motivation
+
+Some apps want to use different templates based on the steering wheel location. As an example TEXT_WITH_BUTTONS_AND_GRAPHIC works great for left hand drives as buttons are closer to the driver. For right hand drives apps would want to use GRAPHIC_WITH_TEXT_AND_BUTTONS. Today apps would use the language/region code and **guess** the location of the steering wheel.
```
Comment (theresalech) [inline proposals/NNNN-steering-wheel-location.md]: Please change "left hand drives" to "left handed drivers" and "right hand drives" to "right handed drivers,"

File: proposals/NNNN-steering-wheel-location.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Proposed solution
+
+In order to improve the user experience for right hand drive the app should know the steering wheel location. This information can be provided to the app right after the apps registration on the head unit. This newly added parameter should be optional for backwards compatibility. The type of the parameter should be an enum. Following values should be allowed:
```
Comment (theresalech) [inline proposals/NNNN-steering-wheel-location.md]: Please change "right hand drive" to "right handed drivers," and "apps registration" to "app's registration"

File: proposals/NNNN-steering-wheel-location.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+- `null`: Either the head unit is of an old version and/or the steering wheel location is not known.
+- `.LEFT`: The steering wheel location is on the left hand side.
+- `.RIGHT`: The steering wheel location is on the right hand side.
+- `.CENTER`: The steering wheel location is centered.
+- `.NONE`: The vehicle does not contain a steering wheel (e.g. autonomous. different to `null` the app would definitely know that no steering wheel exist).
```
Comment (theresalech) [inline proposals/NNNN-steering-wheel-location.md]: Please change "exist" to "exists"

**Comment by kshala-ford** [comment]
@theresalech just removed the grammar errors you mentioned.

**Pull request merged**